tools/stats-definition-exporter/README.md

```diff
@@ -0,0 +1,83 @@
+# Sync Gateway Prometheus Stat Definition Exporter
+This tool exports all the Prometheus Sync Gateway stats definitions to a JSON file. The JSON is prettified by using tab indents and new lines.
+
+## Output
+### To File
+By default, this tool will output to a file. named `metrics_metadata.json`. This is outputted in the current working directory.
+
+Use the `-output` flag to change the output location and name.
+
+### To Stdout
+To output to stdout only and not to a fail, use the `-no-file` flag. The `-output` flag will be ignored when this is `true`.
+
+## Logging
+The tool may output to stderr when unexpected problems occur. For example, this maybe due to a stat not being initialized or a stat being nil.
+
+## JSON format
+The JSON is an array of objects in the current format:
+```json
+[
+  {
+    "name": "string",
+    "unit": "string",
+    "labels": ["strings"],
+    "help": "string",
+    "added": "string",
+    "stability": "string",
+    "type": "string"
```

Comment:
There's no mention of `deprecated`, which I'd have expected so we're able to communicate upcoming stat changes/removals as per the doc.

Is the linked doc/spec outdated in this area?